packages/model/src/user.ts

```diff
@@ -5,7 +5,9 @@ import { TeamRole } from './team';
 export const userRole = ['Staff', 'Grantee', 'Guest', 'Hidden'] as const;
 export type Role = typeof userRole[number];
 
-export const userTags = ['CRN Member', 'Alumni Member'] as const;
+export const activeUserTag = 'CRN Member';
+export const inactiveUserTag = 'Alumni Member';
```

Comment:
I've decided to put this in a variable because in case we change how we call an active/inactive user it's easier to propagate the change